timemanagercontrol.py

```diff
@@ -216,6 +244,13 @@ def refreshGuiWithCurrentTime(self, currentTimePosition, sender=None):
         finally:
             self.setPropagateGuiChanges(True)
 
+    def refreshGuiTimeFrameProperties(self):
+        self.guiControl.setTimeFrameType(self.getTimeLayerManager().timeFrameType)
+        # refreshing of horizontal ruler too, because ticks and discrete-or-not may be changed:
+        self.refreshGuiTimeExtents(self.getTimeLayerManager().getProjectTimeExtents())
+        # SHOULD THIS BE DONE VIA SIGNALS?
+        self.guiControl.dock.spinBoxTimeExtent.setValue(self.getTimeLayerManager().timeFrameSize)
```

Comment:
Not signals per se, but the guiControl could have a function to update the spinboxtimextent instead of being updated directly via member access (dock etc)